packages/docs/src/routes/docs/layout.tsx

```diff
@@ -47,15 +51,18 @@ export default component$(() => {
           </ol>
         ) : null}
       </nav>
-      <div class="md:flex gap-12 xl:gap-20 items-stretch content-container">
+      <div class="flex gap-12 xl:gap-20 items-stretch content-container">
```

Comment:
This [PR](https://github.com/BuilderIO/qwik/pull/6094/files) fixed an issue in the css.
I reintroduce the change
```suggestion
      <div class="md:flex gap-12 xl:gap-20 items-stretch content-container">
```

Reply:
I mentioned this in the description:

> I replaced your fix for horizontal scrolling on mobile with min-w-48 on .docs-container to mitigate possible issues on larger screens as well.

Without the `md:`, the issue that caused #6094 should be fixed as well (due to the newly introduced `min-w-48`). Since removing the `md:` provides less cognitive complexity, I decided to do so.

Reply:
Great, thanks. I removed it.